client/lib/olark/index.js

```diff
@@ -151,6 +151,12 @@ const olark = {
 		}
 	},
 
+	fetchUserEligibility() {
```

Comment:
After thinking about this more maybe `fetchUserEligibility` should be named something else since it actually fetches the olark configuration and then updates the olark group and user eligibility. Nothing comes to mind right now